tools/cli/src/opt/kanidm.rs

```diff
@@ -1048,6 +1048,16 @@ pub enum Oauth2Opt {
         #[clap(name = "landing-url")]
         url: Url,
     },
+    /// The image presented on the Kanidm Apps Listing page for a oauth2 resource server.
+    #[clap(name="set-image")]
+    SetImage {
+        #[clap(flatten)]
+        nopt: Named,
+        #[clap(name = "file-path")]
+        path: String,
```

Comment:
Should be a pathbuf (we really shouldn't have used strings here in the past ...)